2022/12/01 22:00:23 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "gii". in C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii-webapp/index.php?r=gii
---
2022/12/01 22:08:46 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "gii". in C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php:486
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(270): CController->missingAction('gii')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('gii')
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('site/gii')
#3 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii-webapp/index.php?r=site/gii
---
2022/12/01 22:08:55 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "gii". in C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii-webapp/index.php?r=gii
---
2022/12/01 22:54:36 [error] [exception.CException] CException: Alias "user.php" is invalid. Make sure it points to an existing directory or file. in C:\xampp\htdocs\yii-1.1-master\framework\YiiBase.php:355
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\gii\generators\crud\CrudCode.php(59): YiiBase::import('user.php', true)
#1 C:\xampp\htdocs\yii-1.1-master\framework\validators\CInlineValidator.php(42): CrudCode->validateModel('model', Array)
#2 C:\xampp\htdocs\yii-1.1-master\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(CrudCode), 'model')
#3 C:\xampp\htdocs\yii-1.1-master\framework\base\CModel.php(159): CValidator->validate(Object(CrudCode), Array)
#4 C:\xampp\htdocs\yii-1.1-master\framework\gii\CCodeGenerator.php(159): CModel->validate()
#5 C:\xampp\htdocs\yii-1.1-master\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#6 C:\xampp\htdocs\yii-1.1-master\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#7 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('index')
#11 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#12 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yii-webapp/index.php?r=gii/crud/index
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=gii/crud/index
---
2022/12/02 02:42:07 [error] [php] include(MasUser.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii-1.1-master\framework\YiiBase.php:463)
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\YiiBase.php(463): include()
#1 unknown(0): autoload()
#2 C:\xampp\htdocs\yii-webapp\protected\components\UserIdentity.php(33): spl_autoload_call()
#3 C:\xampp\htdocs\yii-webapp\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#4 C:\xampp\htdocs\yii-1.1-master\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#5 C:\xampp\htdocs\yii-1.1-master\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#6 C:\xampp\htdocs\yii-1.1-master\framework\base\CModel.php(159): CInlineValidator->validate()
#7 C:\xampp\htdocs\yii-webapp\protected\controllers\SiteController.php(94): LoginForm->validate()
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#9 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(308): CInlineAction->runWithParams()
#10 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(286): SiteController->runAction()
#11 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): SiteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): SiteController->run()
#13 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii-webapp\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-webapp/index.php?r=site/login
in C:\xampp\htdocs\yii-webapp\protected\components\UserIdentity.php (33)
in C:\xampp\htdocs\yii-webapp\protected\models\LoginForm.php (54)
in C:\xampp\htdocs\yii-webapp\protected\controllers\SiteController.php (94)
2022/12/02 02:43:07 [error] [php] include(MasUser.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii-1.1-master\framework\YiiBase.php:463)
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\YiiBase.php(463): include()
#1 unknown(0): autoload()
#2 C:\xampp\htdocs\yii-webapp\protected\components\UserIdentity.php(33): spl_autoload_call()
#3 C:\xampp\htdocs\yii-webapp\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#4 C:\xampp\htdocs\yii-1.1-master\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#5 C:\xampp\htdocs\yii-1.1-master\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#6 C:\xampp\htdocs\yii-1.1-master\framework\base\CModel.php(159): CInlineValidator->validate()
#7 C:\xampp\htdocs\yii-webapp\protected\controllers\SiteController.php(94): LoginForm->validate()
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#9 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(308): CInlineAction->runWithParams()
#10 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(286): SiteController->runAction()
#11 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): SiteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): SiteController->run()
#13 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii-webapp\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-webapp/index.php?r=site/login
in C:\xampp\htdocs\yii-webapp\protected\components\UserIdentity.php (33)
in C:\xampp\htdocs\yii-webapp\protected\models\LoginForm.php (54)
in C:\xampp\htdocs\yii-webapp\protected\controllers\SiteController.php (94)
2022/12/02 02:45:16 [error] [php] include(inc.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii-1.1-master\framework\YiiBase.php:463)
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\YiiBase.php(463): include()
#1 unknown(0): autoload()
#2 C:\xampp\htdocs\yii-webapp\protected\components\UserIdentity.php(34): spl_autoload_call()
#3 C:\xampp\htdocs\yii-webapp\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#4 C:\xampp\htdocs\yii-1.1-master\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#5 C:\xampp\htdocs\yii-1.1-master\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#6 C:\xampp\htdocs\yii-1.1-master\framework\base\CModel.php(159): CInlineValidator->validate()
#7 C:\xampp\htdocs\yii-webapp\protected\controllers\SiteController.php(94): LoginForm->validate()
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#9 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(308): CInlineAction->runWithParams()
#10 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(286): SiteController->runAction()
#11 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): SiteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): SiteController->run()
#13 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii-webapp\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-webapp/index.php?r=site/login
in C:\xampp\htdocs\yii-webapp\protected\components\UserIdentity.php (34)
in C:\xampp\htdocs\yii-webapp\protected\models\LoginForm.php (54)
in C:\xampp\htdocs\yii-webapp\protected\controllers\SiteController.php (94)
2022/12/02 02:46:22 [error] [php] include(inc.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii-1.1-master\framework\YiiBase.php:463)
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\YiiBase.php(463): include()
#1 unknown(0): autoload()
#2 C:\xampp\htdocs\yii-webapp\protected\components\UserIdentity.php(34): spl_autoload_call()
#3 C:\xampp\htdocs\yii-webapp\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#4 C:\xampp\htdocs\yii-1.1-master\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#5 C:\xampp\htdocs\yii-1.1-master\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#6 C:\xampp\htdocs\yii-1.1-master\framework\base\CModel.php(159): CInlineValidator->validate()
#7 C:\xampp\htdocs\yii-webapp\protected\controllers\SiteController.php(94): LoginForm->validate()
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#9 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(308): CInlineAction->runWithParams()
#10 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(286): SiteController->runAction()
#11 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): SiteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): SiteController->run()
#13 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii-webapp\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-webapp/index.php?r=site/login
in C:\xampp\htdocs\yii-webapp\protected\components\UserIdentity.php (34)
in C:\xampp\htdocs\yii-webapp\protected\models\LoginForm.php (54)
in C:\xampp\htdocs\yii-webapp\protected\controllers\SiteController.php (94)
2022/12/02 03:17:40 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/admin')
#9 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/admin
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=user/index
---
2022/12/02 03:18:56 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/admin')
#9 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/admin
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=user/view&id=3
---
2022/12/02 03:20:32 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('delete')
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/delete')
#9 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/delete&id=3
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=site/login
---
2022/12/02 03:20:50 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('delete')
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/delete')
#9 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/delete&id=3
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=user/view&id=3
---
2022/12/02 03:21:16 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/admin')
#9 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/admin
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=user/view&id=3
---
2022/12/02 03:21:58 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/admin')
#9 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/admin
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=user/view&id=3
---
2022/12/02 03:23:38 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/admin')
#9 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/admin
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=user
---
2022/12/02 03:23:58 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/admin')
#9 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/admin
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=user/index
---
2022/12/02 03:26:10 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/admin')
#9 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/admin
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=user/create
---
2022/12/02 03:27:34 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/admin')
#9 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/admin
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=user/index
---
2022/12/02 03:30:22 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "hapus". in C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php:486
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(270): CController->missingAction('hapus')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('hapus')
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/hapus')
#3 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/hapus
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=user/view&id=3
---
2022/12/02 06:54:34 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "hapus". in C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php:486
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(270): CController->missingAction('hapus')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('hapus')
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/hapus')
#3 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/hapus
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=user/view&id=3
---
2022/12/02 06:54:43 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/admin')
#9 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/admin
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=user
---
2022/12/03 04:32:04 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/admin')
#9 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/admin
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=user/index
---
2022/12/03 04:35:00 [error] [exception.ParseError] ParseError: syntax error, unexpected 'the' (T_STRING) in C:\xampp\htdocs\yii-webapp\protected\controllers\UserController.php:67
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(276): CWebApplication->createController('index/')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#2 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#3 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#4 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/index
HTTP_REFERER=https://localhost/yii-webapp/index.php
---
2022/12/03 04:35:09 [error] [exception.ParseError] ParseError: syntax error, unexpected 'the' (T_STRING) in C:\xampp\htdocs\yii-webapp\protected\controllers\UserController.php:67
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(276): CWebApplication->createController('index/')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#2 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#3 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#4 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/index
HTTP_REFERER=https://localhost/yii-webapp/index.php
---
2022/12/03 04:36:37 [error] [exception.ParseError] ParseError: syntax error, unexpected 'the' (T_STRING) in C:\xampp\htdocs\yii-webapp\protected\controllers\UserController.php:67
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(276): CWebApplication->createController('index/')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#2 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#3 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#4 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/index
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=site/login
---
2022/12/03 04:37:07 [error] [exception.ParseError] ParseError: syntax error, unexpected 'the' (T_STRING) in C:\xampp\htdocs\yii-webapp\protected\controllers\UserController.php:67
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(276): CWebApplication->createController('index/')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#2 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#3 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#4 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/index
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=site/login
---
2022/12/03 04:37:07 [error] [exception.ParseError] ParseError: syntax error, unexpected 'the' (T_STRING) in C:\xampp\htdocs\yii-webapp\protected\controllers\UserController.php:67
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(276): CWebApplication->createController('index/')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#2 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#3 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#4 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/index
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=site/login
---
2022/12/03 04:37:45 [error] [exception.ParseError] ParseError: syntax error, unexpected 'the' (T_STRING) in C:\xampp\htdocs\yii-webapp\protected\controllers\UserController.php:67
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(276): CWebApplication->createController('index/')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#2 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#3 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#4 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/index
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=site/login
---
2022/12/03 04:39:44 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user2/admin')
#9 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii-webapp/index.php?r=user2/admin
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=site/login
---
2022/12/03 04:40:35 [error] [exception.ParseError] ParseError: syntax error, unexpected 'the' (T_STRING) in C:\xampp\htdocs\yii-webapp\protected\controllers\UserController.php:67
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(276): CWebApplication->createController('index/')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#2 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#3 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#4 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/index
HTTP_REFERER=https://localhost/yii-webapp/index.php
---
2022/12/03 04:42:23 [error] [exception.ParseError] ParseError: syntax error, unexpected 'the' (T_STRING) in C:\xampp\htdocs\yii-webapp\protected\controllers\UserController.php:67
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(276): CWebApplication->createController('index/')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#2 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#3 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#4 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/index
HTTP_REFERER=https://localhost/yii-webapp/index.php
---
2022/12/03 04:44:36 [error] [exception.ParseError] ParseError: syntax error, unexpected 'the' (T_STRING) in C:\xampp\htdocs\yii-webapp\protected\controllers\UserController.php:67
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(276): CWebApplication->createController('index/')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#2 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#3 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#4 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/index
HTTP_REFERER=https://localhost/yii-webapp/index.php
---
2022/12/03 04:45:40 [error] [exception.ParseError] ParseError: syntax error, unexpected 'the' (T_STRING) in C:\xampp\htdocs\yii-webapp\protected\controllers\UserController.php:67
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(276): CWebApplication->createController('index/')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#2 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#3 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#4 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/index
HTTP_REFERER=https://localhost/yii-webapp/index.php
---
2022/12/03 04:45:41 [error] [exception.ParseError] ParseError: syntax error, unexpected 'the' (T_STRING) in C:\xampp\htdocs\yii-webapp\protected\controllers\UserController.php:67
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(276): CWebApplication->createController('index/')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#2 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#3 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#4 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/index
HTTP_REFERER=https://localhost/yii-webapp/index.php
---
2022/12/03 04:45:42 [error] [exception.ParseError] ParseError: syntax error, unexpected 'the' (T_STRING) in C:\xampp\htdocs\yii-webapp\protected\controllers\UserController.php:67
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(276): CWebApplication->createController('index/')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#2 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#3 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#4 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/index
HTTP_REFERER=https://localhost/yii-webapp/index.php
---
2022/12/03 04:48:37 [error] [exception.ParseError] ParseError: syntax error, unexpected 'the' (T_STRING) in C:\xampp\htdocs\yii-webapp\protected\controllers\UserController.php:67
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(276): CWebApplication->createController('index/')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#2 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#3 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#4 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/index
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=site/login
---
2022/12/03 04:49:17 [error] [exception.ParseError] ParseError: syntax error, unexpected 'the' (T_STRING) in C:\xampp\htdocs\yii-webapp\protected\controllers\UserController.php:67
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(276): CWebApplication->createController('')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#2 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#3 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#4 {main}
REQUEST_URI=/yii-webapp/index.php?r=user
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=site/index
---
2022/12/03 04:49:43 [error] [exception.ParseError] ParseError: syntax error, unexpected 'the' (T_STRING) in C:\xampp\htdocs\yii-webapp\protected\controllers\UserController.php:67
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(276): CWebApplication->createController('view/')
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/view')
#2 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#3 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#4 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/view
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=site/index
---
2022/12/03 04:54:53 [error] [exception.CException] CException: User2Controller cannot find the requested view "admin". in C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(785): CController->renderPartial('admin', Array, true)
#1 C:\xampp\htdocs\yii-webapp\protected\controllers\User2Controller.php(142): CController->render('admin', Array)
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\actions\CInlineAction.php(49): User2Controller->actionAdmin()
#3 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('admin')
#12 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user2/admin')
#13 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii-webapp/index.php?r=user2/admin
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=site/login
---
2022/12/03 04:55:28 [error] [exception.CException] CException: User2Controller cannot find the requested view "admin". in C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(785): CController->renderPartial('admin', Array, true)
#1 C:\xampp\htdocs\yii-webapp\protected\controllers\User2Controller.php(142): CController->render('admin', Array)
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\actions\CInlineAction.php(49): User2Controller->actionAdmin()
#3 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('admin')
#12 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user2/admin')
#13 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii-webapp/index.php?r=user2/admin
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=user2/create
---
2022/12/03 05:00:19 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "user/index". in C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#1 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii-webapp/index.php?r=user/index
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=site/login
---
2022/12/03 05:01:32 [error] [php] include(AMF.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii-1.1-master\framework\YiiBase.php:463)
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\YiiBase.php(463): include()
#1 unknown(0): autoload()
#2 C:\xampp\htdocs\yii-webapp\protected\views\layouts\main.php(52): spl_autoload_call()
#3 C:\xampp\htdocs\yii-1.1-master\framework\web\CBaseController.php(126): require()
#4 C:\xampp\htdocs\yii-1.1-master\framework\web\CBaseController.php(95): SiteController->renderInternal()
#5 C:\xampp\htdocs\yii-1.1-master\framework\web\widgets\CContentDecorator.php(76): SiteController->renderFile()
#6 C:\xampp\htdocs\yii-1.1-master\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate()
#7 C:\xampp\htdocs\yii-1.1-master\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput()
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\CBaseController.php(215): CContentDecorator->run()
#9 C:\xampp\htdocs\yii-1.1-master\framework\web\CBaseController.php(309): SiteController->endWidget()
#10 C:\xampp\htdocs\yii-webapp\protected\views\layouts\column1.php(6): SiteController->endContent()
#11 C:\xampp\htdocs\yii-1.1-master\framework\web\CBaseController.php(126): require()
#12 C:\xampp\htdocs\yii-1.1-master\framework\web\CBaseController.php(95): SiteController->renderInternal()
#13 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(787): SiteController->renderFile()
#14 C:\xampp\htdocs\yii-webapp\protected\controllers\SiteController.php(98): SiteController->render()
#15 C:\xampp\htdocs\yii-1.1-master\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#16 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(308): CInlineAction->runWithParams()
#17 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(286): SiteController->runAction()
#18 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): SiteController->runActionWithFilters()
#19 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): SiteController->run()
#20 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController()
#21 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#22 C:\xampp\htdocs\yii-webapp\index.php(13): CWebApplication->run()
REQUEST_URI=/yii-webapp/index.php?r=site/login
in C:\xampp\htdocs\yii-webapp\protected\views\layouts\main.php (52)
in C:\xampp\htdocs\yii-webapp\protected\views\layouts\column1.php (6)
in C:\xampp\htdocs\yii-webapp\protected\controllers\SiteController.php (98)
2022/12/03 05:01:46 [error] [exception.Error] Error: Call to undefined function AMF() in C:\xampp\htdocs\yii-webapp\protected\views\layouts\main.php:52
Stack trace:
#0 C:\xampp\htdocs\yii-1.1-master\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii-1.1-master\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#2 C:\xampp\htdocs\yii-1.1-master\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii-1.1-master\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#4 C:\xampp\htdocs\yii-1.1-master\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#5 C:\xampp\htdocs\yii-1.1-master\framework\web\CBaseController.php(215): COutputProcessor->run()
#6 C:\xampp\htdocs\yii-1.1-master\framework\web\CBaseController.php(309): CBaseController->endWidget('CContentDecorat...')
#7 C:\xampp\htdocs\yii-webapp\protected\views\layouts\column1.php(6): CBaseController->endContent()
#8 C:\xampp\htdocs\yii-1.1-master\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#9 C:\xampp\htdocs\yii-1.1-master\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#10 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(787): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#11 C:\xampp\htdocs\yii-webapp\protected\controllers\SiteController.php(98): CController->render('login', Array)
#12 C:\xampp\htdocs\yii-1.1-master\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#13 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#14 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#15 C:\xampp\htdocs\yii-1.1-master\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(282): CController->run('login')
#17 C:\xampp\htdocs\yii-1.1-master\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#18 C:\xampp\htdocs\yii-1.1-master\framework\base\CApplication.php(185): CWebApplication->processRequest()
#19 C:\xampp\htdocs\yii-webapp\index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/yii-webapp/index.php?r=site/login
HTTP_REFERER=https://localhost/yii-webapp/index.php?r=site/login
---
